services/ingestion/etl/budg/xls/src/lib/transform.js

```diff
@@ -30,8 +30,12 @@ export default record => {
   const locationArray = record['Participating countries']
     .split(',')
     .map(country => ({
-      country_name: null,
       country_code: country,
+      region: null,
```

Comment:
Those defaults should be `''`, not `null`, shouldn't they?

Reply:
Yes, that was the point of my comment, we could update it in follow-up when adding tests to the ETL